test/runtests.jl

```diff
@@ -14,6 +14,14 @@ end
 @testset "ClimateMachine" begin
     all_tests = isempty(ARGS) || "all" in ARGS ? true : false
 
+    function has_submodule(sm)
+        any(ARGS) do a
+            a == sm && return true
+            first(split(a, '/')) == sm && return true
```

Comment:
Not a blocker, but this string parsing won't work on Windows

Reply:
There's quite a bit of hard-coded `/`'s in the tests at the moment, though. We can change that later if we need

Reply:
I think it's ok here as it's just specifying a label and not a full path

```Numerics/DGMethods``` -> run the subset of tests under under the numerics group

maybe there is a less confusing prefix we can use to define test groupings